less/dropdowns.less

```diff
@@ -198,12 +198,12 @@
       left: 0;
       width: 100vw;
       height: 100vh;
+      border: 0;
+      border-radius: 0;
+      box-shadow: none;
       max-width: none;
-      margin: 0;
       overflow-y: auto;
-      background-color: #fff;
       animation: listSlideOver 0.15s ease-out forwards;
-      z-index: 1;
```

Comment:
Removing these CSS properties means we're using the default properties from the dropdown modal CSS class